### PR TITLE
completed code challenge linked list

### PR DESCRIPTION
**Pull request by greasonek**

### CODE REVIEW

**Comment by greasonek** [comment]
- [x ] Top-level README “Table of Contents” is updated
 - [x ] README for this challenge is complete
       - [ x] Summary, Description, Approach & Efficiency, Solution
       - [ x] Picture of whiteboard
       - [ x] Link to code
 - x[ ] Feature tasks for this challenge are completed
 - [x ] Unit tests written and passing
       - [x ] “Happy Path” - Expected outcome
       - [x ] Expected failure
       - [x ] Edge Case (if applicable/obvious)

**Comment by greasonek** [comment]
[x ] Top-level README “Table of Contents” is updated
[x ] README for this challenge is complete
- [ x] Summary, Description, Approach & Efficiency, Solution
- [ x] Picture of whiteboard
- [ x] Link to code
x[ ] Feature tasks for this challenge are completed
[x ] Unit tests written and passing
- [x ] “Happy Path” - Expected outcome
- [x ] Expected failure
- [x ] Edge Case (if applicable/obvious)

File: javascript/linked-list/__tests__/linked-list.test.js [hunk 1]
```diff
@@ -1,10 +1,33 @@
-'use strict';
+"use strict";
 
-// Require our linked list implementation
-const LinkedList = require('../index');
+const { LinkedList } = require("./linkedList.js");
```
Comment (lizkavalski) [inline javascript/linked-list/__tests__/linked-list.test.js:3]: change `"./linkedList.js"` to `../index.js`
